src/format/ArticleSpecial.ts

```diff
@@ -1,4 +1,3 @@
-export enum ArticleSpecial {
-	SpecialReport = 5,
-	Labs = 6,
-}
+type ArticleSpecial = 'SpecialReport' | 'SpecialLabs';
```

Comment:
I'm not 100% sure how up to date [these Frontend types](https://github.com/guardian/frontend/blob/oliver/press-aus-weekend/common/app/model/meta.scala#L288) are but do we want to follow the pattern there and have `SpecialReportTheme` and then maybe `Labs` (or perhaps `LabsTheme`?).

Reply:
@OllysCoding, when this came up when we were working on formats-explorer, I think you had some thoughts relating to these categories?